document.txt

```diff
@@ -0,0 +1,8 @@
+This is an important document 
+-------------
+
+Here's an important list:
+1) Important item

```

Comment:
Another test.

---
In reply to: [54177272](https://github.com/kivancmuslu/CFTest/pull/2#discussion_r54177272) [](ancestors = 54177272,54176679,54169441,54165983,54165527)